### ISSUE TITLE
Is it possible to modify the entity of existing experiments? [Q]

### ISSUE THREAD

**Issue by Erosinho13** [wandb/wandb#5089]
My account belongs to three teams. A group of experiments from one project accidentally fall into the entity of the wrong team. How can I modify the entity of existing experiments? Moreover, after I eventually change the entity, I still need to be able programmatically to access the information of the runs. Thank you

**Comment by Erosinho13** [comment]
Yes, I am trying to do exactly what you said. My wandb username is erosinho. I would like to move some experiments from the team fl_polito_unipd to the team fedavg_semseg

**Comment by billmdevs** [comment]
Hi @Erosinho13 thank you for your patience. Could you let me know what projects you would like to move across entities? Do you want to move all the projects?

**Comment by Erosinho13** [comment]
Thank you for your interest in my issue.
Why do you need the names of the projects? Are you going to do it by yourselves? I would prefer to do it by myself. Isn't it possible to add an option on the wandb interface?
I want to move all the runs belonging to **some** specific projects from one entity to the projects with the same name but belonging to the other entity, **without** modifying the other projects

**Comment by nate-wandb** [comment]
Hi @Erosinho13, sorry for the delay on this. Could you elaborate on this: "without modifying the other projects"?

On the runs table you have the option to select multiple runs and then can click the move option at the top to move them to another entity/project. The runs will no longer exists in the original project though and this isn't a copy function. 

Thank you,
Nate

**Comment by nate-wandb** [comment]
Hi @Erosinho13, I wanted to follow up and see if you were still looking for help with this?
